### ISSUE TITLE
Composant carte, outil de dessin "rectangle" non fonctionnel

### ISSUE THREAD

**Issue by jpm-cbna** [PnX-SI/GeoNature#1802]
**Version**
develop

**Description du bug**
Dans le module Synthese, sur la carte, l'outil de dessin "rectangle" fonctionne mal. Lorsqu'on sélectionne l'outil et que l'on commence à tracer un rectangle, celui-ci reste tout petit.
RAS pour polygone et cercle

A priori, le problème vient du passage à Angular 12 qui a activé le mode stricte de Javascript.
Le ticket [#1005](https://github.com/Leaflet/Leaflet.draw/issues/1005#issuecomment-753758400) sur Leaflet.draw indique que la PR [#1002](https://github.com/Leaflet/Leaflet.draw/pull/1002) résoud le problème.

En modifiant le code dans `node_modules/dist/leaflet.draw-src.js` puis en le [minimifiant](https://jscompress.com/) et collant le résultat dans `node_modules/dist/leaflet.draw.js`, le problème est bien réglé...
La question est de savoir comment récupérer cette correction pour GeoNature ?

**Comportement attendue**
Un rectangle devrait être dessiner sur la carte.

**Comment reproduire**
Installer la version "develop" de GeoNature, se rendre dans le module "Synthese" et essayer de dessiner un rectangle avec l'outil disponible sur la carte.

**Logs**
Extrait du fichier ``/var/log/geonature.log`` dans le cas d’une erreur 500.

**Comment by camillemonchicourt** [comment]
Oui le soucis est que Leaflet.draw ne semble plus maintenu.
Il faudrait songer à la remplacer.
Peut-être par https://github.com/geoman-io/leaflet-geoman ?

**Comment by jpm-cbna** [comment]
Oui, `leaflet-geoman` a l'air d'avoir les mêmes fonctionnalités et elle est maintenu.
Par contre, pour la v2.10 de GN et résoudre le bug en question, je pencherai plutôt pour intégrer la bibliothèque `leaflet.draw` directement dans GeoNature comme l'a proposé @TheoLechemia . Nous corrigerons ainsi le bug (très simple) et pourrons continuer à l'utiliser le temps d'une migration vers `leaflet-geoman`.

**Comment by jacquesfize** [comment]
Fonctionne en 2.15.3
